### ISSUE TITLE
geom_ribbon shows no value if min and max values are equal - shouldn't default be to show a line

### ISSUE THREAD

**Issue by morrisseyj** [tidyverse/ggplot2#5659]
When using geom_ribbon, with equal values for min and max, I would expect the output to be a line. Currently nothing is graphed. 

Example below: 

```
#Load libraries
library("tidyverse")

#Create example data
data <- data.frame("min" = 1:5, 
                   "max" = c(1, 2, 5, 7, 9), 
                   "time" = 1:5) 
#Plot
data %>%
  ggplot(aes(x = time, ymin = min, ymax = max)) + 
  geom_ribbon()
```

![ribbon](https://github.com/tidyverse/ggplot2/assets/55301804/401b26b2-8cc3-48b5-a03a-92221d3714a2)

To achieve the desired behaviour one has to create a midpoint variable:

```
#Create new midpoint variable
data_edit <- data %>%
  mutate(midpoint = (min + max)/2) 

#Plot
data_edit %>%
  ggplot(aes(x = time, y = midpoint, ymin = min, ymax = max)) + 
  geom_line() +
  geom_ribbon()

```

![ribbon2](https://github.com/tidyverse/ggplot2/assets/55301804/50e64009-a1f1-4860-9580-e546ddb0eedb)

While i appreciate the ribbon is an area and area is zero where values are equal, this seems a common enough case (plotting something about things are known now, but will vary in the future, that either it should be the default or there should be an option to generate a line should values be equal. 

Apologies if this already exists and I have missed it in the documentation.

**Comment by teunbrand** [comment]
I think you can just achieve this by setting a line colour with `geom_ribbon(colour = 'black')` for examplel.

**Comment by morrisseyj** [comment]
@teunbrand Yes, that works. Apologies. That is straightforward. I'll close this.
